src/pages-and-resources/discussions/app-config-form/AppConfigForm.jsx

```diff
@@ -49,10 +59,20 @@ function AppConfigForm({
   }, [selectedAppId, routeAppId, status]);
 
   // This is a callback that gets called after the form has been submitted successfully.
-  const handleSubmit = useCallback((values) => {
-    // Note that when this action succeeds, we redirect to pagesAndResurcesPath in the thunk.
-    dispatch(saveAppConfig(courseId, selectedAppId, values, pagesAndResourcesPath));
-  }, [courseId, selectedAppId, courseId]);
+  const handleSubmit = useCallback((values, actions) => {
+    actions.setSubmitting(false);
```

Comment:
Have you checked for ripples of this effect? The form is not getting submitted for EDX discussions because of this change and an error is occurring `Cannot read property of setSubmitting. ` can you please check it. one observation is that in lti config form formik context is being used while in legacy form no such thing is available. so you may need to check it otherwise it will break existing code.

Reply:
Thanks for catching this; in fact, both forms use Formik but the legacy one was throwing away the `values` parameter: I've moved the call to `setSubmitting` into each of the forms, which makes things easier to understand.
This should work but I'm unable to test it locally because attempting to configure the legacy provider raises `TypeError: divideDiscussionIds is undefined`...

Reply:
can you please check with the master branch if that issue is not coming there then it must also be the ripple of some changes done in this PR.